src/controllers/features/update-acsp/updateIndexController.ts

```diff
@@ -0,0 +1,24 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
As this controller is part of /features/update-acsp/, I don't think we need to name it as **update**IndexController. You can make it indexController